### PR TITLE
Bug 1802584 - Announce "Find in page" edit box when using Talkback

### PR DESCRIPTION
**Pull request by DreVla**
When using Talkback and navigating to the find in page function upon opening the edit box, "Find in page" is announced. Previously it seemed like the focus was jumping around the `FindInPageBar` ([video](https://bugzilla.mozilla.org/show_bug.cgi?id=1802584#c6)). One solution was giving up the view stub `stubFindInPage` and after some consideration this was the approach. Giving it up seems to have fixed the issue on all phones and android versions I could test on.

| Nexus A 5.1.1 | OnePlus 6 A11 |
| ------------- | ------------- |
| <video src = "https://user-images.githubusercontent.com/32488956/225360527-df4ee65a-b372-4671-ae47-a44de791b3f4.mp4"> | <video src = "https://user-images.githubusercontent.com/32488956/225360542-1ac92049-c971-43de-937e-faa3f00ac945.mp4"> |

| Pixel 3 A12 | OnePlus 8T A13 |
| ------------- | ------------- |
| <video src = "https://user-images.githubusercontent.com/32488956/225360573-8647816f-a112-4815-b04f-0cb99ee50831.mp4"> | <video src = "https://user-images.githubusercontent.com/32488956/225360588-dc62a86a-94b3-4205-ae68-b29a988056e9.mp4"> |

### Pull Request checklist

- [X] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [X] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [X] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1802584

### CODE REVIEW

**Review by Mugurell — COMMENTED** [review]
Generally lgtm, some small nits.
But I do see the issue still reproducing on a Pixel 7 with Android 13 while indeed on a Samsung A51 with Android 12 the issue disappeared.


https://user-images.githubusercontent.com/11428869/225653255-d03293c0-2a6f-4e34-b13e-d7758f93a54b.mp4

**Comment by DreVla** [comment]
> Generally lgtm, some small nits. But I do see the issue still reproducing on a Pixel 7 with Android 13 while indeed on a Samsung A51 with Android 12 the issue disappeared.

Seems like I missed some accessibility tags for the edit text. I added them and for me on a OnePlus 8T with A13 the issue is not reproducible anymore.

File: fenix/app/src/main/java/org/mozilla/fenix/components/FindInPageIntegration.kt [hunk 1]
```diff
@@ -30,19 +31,38 @@ import org.mozilla.fenix.components.FindInPageIntegration.ToolbarInfo
 class FindInPageIntegration(
     private val store: BrowserStore,
     private val sessionId: String? = null,
-    stub: ViewStub,
+    private val view: FindInPageBar,
     private val engineView: EngineView,
     private val toolbarInfo: ToolbarInfo,
-) : InflationAwareFeature(stub) {
```
Comment (jonalmeida) [inline fenix/app/src/main/java/org/mozilla/fenix/components/FindInPageIntegration.kt:36]: Can you also remove `InflationAwareFeature` as FIP was the only feature using it and we are not tempted to use it again? 😄

Comment (jonalmeida) [reply fenix/app/src/main/java/org/mozilla/fenix/components/FindInPageIntegration.kt:36]: You can also close [this issue](https://bugzilla.mozilla.org/show_bug.cgi?id=1794676) as 'WONTFIX' if you do the above.

**Review by jonalmeida — APPROVED** [review]
LGTM!

Before landing, please add a space between the commit title and the commit description. If not, the title and description get squashed into one and look like one very long line:

Screenshot 2023-04-10 at 1 32 15 PM


If you are interested in trying it out, I use a variation of [this commit message template](https://gist.github.com/adeekshith/cd4c95a064977cdc6c50) to meet our template requirements. 

My variation is here: https://github.com/jonalmeida/dotfiles/blob/master/.gitmessage
You can apply it for only `firefox-android` if you remove the `--global` from the instructions in the original template.

**Pull request merged**